tests/jballerina-unit-test/build.gradle

```diff
@@ -48,7 +48,7 @@ test {
     systemProperty "java.util.logging.config.file", "src/test/resources/logging.properties"
     systemProperty "enableJBallerinaTests", "true"
 
-    jvmArgs = ['-Xms512m', '-Xmx3g', '--add-opens=java.base/java.util=ALL-UNNAMED',
+    jvmArgs = ['--add-opens=java.base/java.util=ALL-UNNAMED',
```

Comment:
Without removing these args, shall we just reduce the values so that they are definite.

Reply:
If we need to reduce, we need to do that in all tests. I think 3g here is too high. IMO we need to allow gradle to decide on allocation of memory for each tests.

Reply:
If we do that, won't it change from machine to machine? Like when we run locally in a 16 GB RAM machine the max heap size will be high, and in a 7 GB Github runner the max heap size will be low.